ci/build.sh

```diff
@@ -35,6 +35,8 @@ COMPONENTS['core']='OPENVDB_BUILD_CORE'
 COMPONENTS['python']='OPENVDB_BUILD_PYTHON_MODULE'
 COMPONENTS['test']='OPENVDB_BUILD_UNITTESTS'
 COMPONENTS['bin']='OPENVDB_BUILD_BINARIES'
+COMPONENTS['view']='DOPENVDB_BUILD_VDB_VIEW'
+COMPONENTS['render']='DOPENVDB_BUILD_VDB_RENDER'
```

Comment:
Minor mistake here - remove the preceding `D`, otherwise it's being passed as `-DDOPENVDB_BUILD_VDB_VIEW`

Reply:
Good catch, thanks